docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -1,14 +1,595 @@
+
+    * Files.Read.All
+    * offline_access
+    * openid
+    * profile
+
+    > [!NOTE]
+    > The `User.Read` permission may already be listed by default. It is a good practice not to ask for permissions that are not needed, so we recommend that you uncheck the box for this permission if your add-in does not actually need it.
+
+1. Select the check box for each permission as it appears. After selecting the permissions that your add-in needs, select the **Add permissions** button at the bottom of the panel.
+
+1. On the same page, choose the **Grant admin consent for [tenant name]** button, and then select **Yes** for the confirmation that appears.
+
+## Configure the solution
+
+1. In **Visual Studio**, right-click the top node in **Solution Explorer** (the Solution node, not either of the project nodes), and then select **Set startup projects**.
+
+1. Under **Common Properties**, select **Startup Project**, and then **Multiple startup projects**. Ensure that the **Action** for both projects is set to **Start**, and that the project that ends in "...WebAPI" is listed first.
+
+1. Still in **Solution Explorer**, choose the **Office-Add-in-Microsoft-Graph-ASPNETWebAPI** project. In **Properties**, ensure **SSL Enabled** is **True**. Verify that the **SSL URL** is `http://localhost:44355/`.
+
+1. In web.config, use the values that you copied in earlier. Set both the **ida:ClientID** and the **ida:Audience** to your **Application (client) ID**, and set **ida:Password** to your client secret.
+
+    > [!NOTE]
+    > The **Application (client) ID** is the "audience" value when other applications, such as the Office host application (e.g., PowerPoint, Word, Excel), seek authorized access to the application. It is also the "client ID" of the application when it, in turn, seeks authorized access to Microsoft Graph.
+
+1. In the add-in project, open the add-in manifest file “Office-Add-in-ASPNET-SSO.xml” and then scroll to the bottom of the file. Just above the end `</VersionOverrides>` tag, you'll find the following markup:
+
+    ```xml
+    <WebApplicationInfo>
+      <Id>$application_GUID here$</Id>
+      <Resource>api://localhost:44355/$application_GUID here$</Resource>
+      <Scopes>
+          <Scope>Files.Read.All</Scope>
+          <Scope>offline_access</Scope>
+          <Scope>openid</Scope>
+          <Scope>profile</Scope>
+      </Scopes>
+    </WebApplicationInfo>
+    ```
+
+1. Replace the placeholder “$application_GUID here$” *in both places* in the markup with the Application ID that you copied when you registered your add-in. The "$" signs are not part of the ID, so do not include them. This is the same ID you used in for the ClientID and Audience in the web.config.
```

Comment:
In later steps, there's an explicit reminder to save the file, but not for Web.Config or Office-Add-in-ASPNET-SSO.xml